pkg/daemon/update.go

```diff
@@ -184,44 +184,21 @@ func (dn *Daemon) reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) *stri
 			// check if the prior config is empty and that this is the first time running.
 			// if so, the SSHKey from the cluster config and user "core" must be added to machine config.
 			if len(oldIgn.Passwd.Users) == 0 && len(newIgn.Passwd.Users) == 1 {
```

Comment:
Offhand, it feels like this "first time running" is actually just for transitioning clusters from *before* this PR to clusters with it (or in the future to support re-installing SSH keys if they've been removed).  Freshly installed clusters *with* this PR and a SSH key supplied should always have `len(oldIgn.Passwd.Users)` right?

Reply:
Nope the first time running part is used from my first SSH PR where we are passing the SSH Keys from the cluster config thru to the MCD.

Reply:
I cleaned up this section in #293